net/lp_rpc.proto

```diff
@@ -20,12 +20,55 @@ message TranscoderRequest {
   bytes sig   = 2;
 }
 
+/*
+  OSInfo needed to negotiate storages that will be used.
+  It carries info needed to write to the storage.
+*/
+message OSInfo {
+
+  enum Storage {
+    DIRECT = 0;
+    LOCAL = 1;
```

Comment:
What's the difference between direct and local?

Reply:
direct not used, added here per you issue, for future negotiation.